packages/components/statistic/src/statistic-custom.ts

```diff
@@ -0,0 +1,11 @@
+import { computed } from 'vue'
+import { useNamespace } from '@element-plus/hooks'
+export function useStatisticCustomStyle() {
```

Comment:
## Suggest

this file doesn't appear to be used anywhere. how about deleting it?

Reply:
OK, sure